docs/metrics.adoc

```diff
@@ -3,22 +3,39 @@
 Metrics is another service provided by Core and no further dependencies are required for its utilization.
 It provides registration of default metrics for the application such as App and Device information as well as supplying these services to other modules of the SDK.
 
+== Usage
+With an instance of the `MetricsService` class, it is possible to send default (app and device) as well as custom metrics to an https://github.com/aerogear/aerogear-app-metrics[Aerogear App Metrics] instance.
+
 [source, javascript]
 ----
-import { MetricsService } from '@aerogear/core';
+import { Metrics, MetricsService } from '@aerogear/core';
 import configuration from '../mobile-services.json';
+
+// Instance of MetricsService
 const metrics = new MetricsService(configuration);
-metrics.sendAppAndDeviceMetrics().then(() => {
-  console.info("Metrics registered successfully");
-});
+
+// Sending default metrics
+metrics.sendAppAndDeviceMetrics()
+  .then(() => {
+    console.info("Metrics registered successfully");
+  });
+
+// Sending additional custom metrics
+const customMetrics: Metrics[] = gatherCustomMetrics();
+metrics.publish("my-custom-type", customMetrics)
+  .then(() => {
+    console.info("Custom metrics registered successfully");
+  });
 ----
 
-== App metrics record fields
+== Default metrics
+
+The default metrics are always sent alongside any custom metrics. They are composed by the following fields:
```

Comment:
Also referring to custom metrics here, could mention that they're sent alongside all metrics records :+1: